postgresql/examples/postgres.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgresql
+spec:
+  template:
+    spec:
+      serviceAccountName: <service-account-with-vault-access>
```

Comment:
I think those can be a Makefile parameter- is there a reason to kustomize- patch it every time?